weather_api/views.py

```diff
@@ -0,0 +1,40 @@
+import os
+import requests
+
+from datetime import datetime
+from django.contrib import messages
+from django.shortcuts import render
+
+from .forms import CityForm
+
+
+def search(request):
+    form = CityForm()
+    city_weather = {}
+
+    if request.method == "POST":
+
+        API_KEY = os.getenv("API_KEY")
+        city_name = request.POST.get("place")
+        url = f"https://api.openweathermap.org/data/2.5/weather?q={city_name}&units=metric&appid={API_KEY}"
+
+        response_json = requests.get(url).json()
+        status = requests.get(url).status_code
+
+        if status == 200:
```

Comment:
To tutaj niepotrzebnie teraz mamy dwa zapytania, wiem, że te na dole kazałem zrobić, ale było na potrzeby pokazania co to jest ten status code bez psucia reszty. Teraz chcielibyśmy wysłać jedno bo tyle potrzebujemy, więc najpierw musisz wysłać request, sprawdzić czy jest spoko, użyj property `ok` o którym mówiliśmy i potem w środku dopiero skorzystaj z `json` żeby mieć dane w formie dicta